### DISCUSSION TITLE
Nugetize and c++/cli

### DISCUSSION THREAD

**Discussion by rfuzzo** [devlooped/nugetizer#123] [Q&A] [answered]
Hej, I am trying to pack a csproj that references a c++/cli vcxproj. 

Is there a way to include the c++/cli dll as a reference in the nuget? (I am not quite sure how to handle x64/x86 version yet, but that's another question)

nugetizer refuses to work (`nugetize` from the commandline) for the same reason that dotnet pack doesn't work:  it uses the wrong msbuild.exe (not the VS one with c++ support)

Is there any way to solve this? Thank you.

**Comment by kzu** [comment, marked as answer]
Perhaps you can provide a small minimalistic repro solution?

**Reply by kzu** [reply]
Closing as answered for now.
